internal/cli/atlas/deployments/options/deployment_opts_connect_string.go

```diff
@@ -17,23 +17,26 @@ package options
 import (
 	"context"
 	"fmt"
+	"strconv"
 
-	"github.com/mongodb/mongodb-atlas-cli/internal/podman"
+	"github.com/containers/podman/v4/libpod/define"
 )
 
 const deploymentTypeLocal = "local"
 
-func (opts *DeploymentOpts) updateFields(c *podman.Container) {
+func (opts *DeploymentOpts) updateFields(c *define.InspectContainerData) {
 	opts.DeploymentType = deploymentTypeLocal
-	opts.MdbVersion = c.Labels["version"]
-	if len(c.Ports) > 0 {
-		opts.Port = c.Ports[0].HostPort
+	opts.MdbVersion = c.Config.Labels["version"]
+	if len(c.HostConfig.PortBindings) > 0 {
+		for _, port := range c.HostConfig.PortBindings {
+			opts.Port, _ = strconv.Atoi(port[0].HostPort)
+		}
```

Comment:
what's in c.HostConfig.PortBindings collection for mongo*, do we have multiple entries? (in which case we're overwriting the opts.Port in this for loop).

Reply:
in some cases (not ours) we can expose multiple ports for the same container, that is why this struct is in this way

Reply:
but in this case the code above will set opts.Port with the last HostPort.
not blocking as our container has only one

Reply:
Probably it should look like this:
```suggestion
		for internalPort, port := range c.HostConfig.PortBindings {
			if strings.HasPrefix(internalPort, strconv.Itoa(InternalMongodPort)) {
				opts.Port, _ = strconv.Atoi(port[0].HostPort)
			}
		}
```

Reply:
I've addressed it, unfortunately reusing the internalMongodPort would require some more coding, so I decided to leave it for a future PR